erdpy/CLI.md

```diff
@@ -203,6 +203,7 @@ optional arguments:
   --pem-index PEM_INDEX                        🔑 the index in the PEM file (default: 0)
   --keyfile KEYFILE                            🔑 a JSON keyfile, if PEM not provided
   --passfile PASSFILE                          🔑 a file containing keyfile's password, if keyfile provided
+  --password PASSWORD                          🔑 the keyfile's password, if keyfile provided
```

Comment:
If we don't want this parameter (see comments below), then we should revert this file as well.

Reply:
Removed